static/js/settings_users.js

```diff
@@ -255,6 +255,15 @@ exports.on_load_success = function (realm_people_data) {
                 }
             },
             success: function () {
+                var user_table_list = list_render.get("users_table_list");
+
+                var user = _.find(user_table_list.data(), function (user) {
+                    return user.email === email;
+                });
+
+                user.is_active = false;
+                user_table_list.render();
+
```

Comment:
I merged all but the last commit.  For this last commit, I can't figure out what problem it solves, but regardless, it seems like something that should be in `people.deactivate` (aka what's called from `server_events_dispatch.js`).

Reply:
it doesn't solve an issue anymore, so that whole block above that can get removed.